### PR TITLE
[meta-oe][warrior] nlohmann-json: upgrade 3.3.0 -> 3.7.0

### PR DESCRIPTION
**Pull request by stkw0**

### CODE REVIEW

File: meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb [hunk 1]
```diff
@@ -6,7 +6,7 @@ LIC_FILES_CHKSUM = "file://LICENSE.MIT;md5=9a8ae1c2d606c432a2aa2e2de15be22a"
 
 SRC_URI = "git://github.com/nlohmann/json.git"
 
-PV = "3.3.0+git${SRCPV}"
+PV = "3.7.0+git${SRCPV}"
```
[the remark this answers is not in the document]
Comment (stkw0) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: Fixed, sry.

[the remark this answers is not in the document]
Comment (stkw0) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: Seems it's not fixed. "Fetcher failure: Unable to find revision 53c3eefa2cf790a7130fed3e13a3be35c2f2ace2 in branch master even from upstream" but it's the right commit

[the remark this answers is not in the document]
Comment (stkw0) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: Now it's all correct :)

File: meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb [hunk 2]
```diff
@@ -2,13 +2,13 @@ SUMMARY = "JSON for modern C++"
 HOMEPAGE = "https://nlohmann.github.io/json/"
 SECTION = "libs"
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE.MIT;md5=9a8ae1c2d606c432a2aa2e2de15be22a"
+LIC_FILES_CHKSUM = "file://LICENSE.MIT;md5=f5f7c71504da070bcf4f090205ce1080"
```
Comment (kraj) [inline meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: what caused the change in checksum here ?

**Comment by kraj** [comment]
merged with 95d95dcb1048a0ee4263ff1c48af962f31fb809f

File: meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb (hunk 2 above)
Comment (stkw0) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: This

> -Copyright (c) 2013-2018 Niels Lohmann
> +Copyright (c) 2013-2019 Niels Lohmann

Comment (kraj) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: usually this should be documented in commit msg

**Comment by kraj** [comment]
this is merged

File: meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb (hunk 2 above)
Comment (stkw0) [reply meta-oe/recipes-devtools/nlohmann-json/nlohmann-json_git.bb]: thanks, I will do it next time.

**Pull request closed without merging**